src/libs/actions/ReportActions.ts

```diff
@@ -33,14 +35,55 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction) {
         return;
     }
 
+    if (keys) {
```

Comment:
This is to handle cases with multiple errors in the same report action, right? 

It may be a good improvement but It is not mentioned in the issue. Why do you think It is our expectation? Do you confirm with the design team or any internal engineer about that?

Reply:
I agree that's the expectation